ts-sdk/whirlpool/src/position.ts

```diff
@@ -85,8 +85,7 @@ function getPositionInBundleAddresses(
  * import { generateKeyPairSigner, createSolanaRpc, devnet } from '@solana/web3.js';
  *
  * const devnetRpc = createSolanaRpc(devnet('https://api.devnet.solana.com'));
- * const wallet = await generateKeyPairSigner();
- * await devnetRpc.requestAirdrop(wallet.address, lamports(1000000000n)).send();
+ * const wallet = await generateKeyPairSigner(); // USE YOUR OWN WALLET
```

Comment:
Think here you can just do address("INSERT_WALLET_ADDRESS"). Since it doesn't have to be a signer